.circleci/config.yml

```diff
@@ -213,7 +213,7 @@ jobs:
             - ~/.cache/pre-commit
       - run:
           name: Run pre-commit
-          command: pre-commit run --all-files --config <<parameters.config_file>>
+          command: ./.husky/pre-commit
       - run:
           name: Run deployment pre-commit
```

Comment:
Can we remove the second pre-commit file & CI step now?

Reply:
:y